synapse/storage/databases/main/cache.py

```diff
@@ -52,6 +53,21 @@
 # As above, but for invalidating room caches on room deletion
 DELETE_ROOM_CACHE_NAME = "dr_cache_fake"
 
+# How long between cache invalidation table cleanups, once we have caught up
+# with the backlog.
+REGULAR_CLEANUP_INTERVAL_MILLISEC = 60 * 60 * 1000
```

Comment:
To align with the rest of the codebase, we probably want to use `_MS`.

Although there is one instance of `_MILLISECONDS` as well which seems better than `_MILLISEC` in any case.

Reply:
units being case-sensitive offends me on some level, now I know it's fairly obvious we're probably not going to be talking about megaseconds, or millisiemens, but just to explain why I wrote it as I did (and to be fair I didn't bother checking for other examples in the code; I will adjust to `MS`) :)